sherpa-onnx/csrc/offline-recognizer-whisper-impl.h

```diff
@@ -23,29 +23,32 @@
 
 namespace sherpa_onnx {
 
-static OfflineRecognitionResult Convert(const OfflineWhisperDecoderResult &src,
-                                        const SymbolTable &sym_table) {
-  OfflineRecognitionResult r;
-  r.tokens.reserve(src.tokens.size());
-
-  std::string text;
-  for (auto i : src.tokens) {
-    if (!sym_table.Contains(i)) {
-      continue;
+class OfflineRecognizerWhisperImpl : public OfflineRecognizerImpl {
+ private:
```

Comment:
Can you move it to the end of the private section for methods in this class?

We usually put the public constructor at the beginning of a class in sherpa-onnx.

(It is a code style issue)

Reply:
Done.  I initially left it at the top so that the diffs are easier to read.